skia/recorder/include/miniz.h

```diff
@@ -0,0 +1,1338 @@
+/* miniz.c 2.1.0 - public domain deflate/inflate, zlib-subset, ZIP reading/writing/appending, PNG writing
```

Comment:
Well, cpp dependency management is a bit of a mess really, and when there's a chance to have simple libraries like this, it's imo kinda of a blessing really. It might slow down the build a bit (this library is sufficiently small luckily), but if it builds, it builds, no linker issues.
In fact, there's a whole "header-only" library for cpp which has tried to get around different build systems that overcomplicate things and are more error-prone. 
We already have a pretty varied setup with some stuff installed through `apt-get`, other things that are `CMake`'d (CMade?), and we also are using the header-only `args.hxx` for argument parsing.
So, overall, I like this more than having to deal with multiple ways of building our deps in the codebase, which translates also in Docker being more difficult to maintain.